### PR TITLE
Fixes Issue #16

### PR DESCRIPTION
**Pull request by celinepark**
Added a summary as an image, with a link to the original Google Doc as a comment in the summary.rst file. Commits were accidentally made from Lindsey's account.

### CODE REVIEW

**Comment by lindseyc** [comment]
@googlebot I consent.

**Comment by celinepark** [comment]
@raskutti Please review

File: ac1/_sources/importing_and_exporting_data/summary.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.. Copyright (C)  Google, Runestone Interactive LLC
+    This work is licensed under the Creative Commons Attribution-ShareAlike 4.0
+    International License. To view a copy of this license, visit
+    http://creativecommons.org/licenses/by-sa/4.0/.
+
+.. Link to summary document: https://docs.google.com/document/d/        1ojHGNfmmn3tshui6ZiFws1mNgbuNQIG53s_gnzVHCQo/edit?usp=sharing
```
Comment (raskutti) [inline ac1/_sources/importing_and_exporting_data/summary.rst]: The long link is a bit harder to click, can you use the short link as per https://github.com/google/applied-computing-series/pull/6/files?

Also, can we replace all previous summary files to use this short link and place the comment below the image?

**Review by raskutti — CHANGES_REQUESTED** [review]
Small change. Image LGTM.

File: ac1/_sources/importing_and_exporting_data/summary.rst (hunk 1 above)
Comment (celinepark) [reply ac1/_sources/importing_and_exporting_data/summary.rst]: We created a new Google Doc for the Module B summaries, so the links will differ a bit from module to module but I can change it to a shorter url

**Comment by celinepark** [comment]
@raskutti Link has been changed

**Review by raskutti — APPROVED** [review]

**Pull request merged**